core/mmu.cpp

```diff
@@ -20,11 +20,31 @@ MMU::~MMU() {
     delete[] memory;
 }
 
+// During mode OAM: CPU cannot access OAM
+// During mode VRAM: CPU cannot access VRAM or OAM
+// During restricted modes, any attempt to read returns $FF, any attempt to write are ignored
+bool MMU::blockedByPPU(u16 address) {
+    u8 stat = memory[0xFF41];
```

Comment:
We should probably pull this out into a constant. If we define `STAT` in the MMU, then the PPU can share it.

Reply:
(by 'this', I mean the address)